Makefile

```
@@ -198,7 +198,7 @@ deps: deps-go generate
 
 .PHONY: generate
 generate: deps-go
-	go generate ./pkg/packagekit/... ./pkg/packaging/...
+	go generate ./pkg/packagekit/... ./pkg/packaging/... ./pkg/osquery/tables/wifi_networks/...
```

```suggestion
	go generate ./pkg/packagekit/... ./pkg/packaging/... ./pkg/osquery/tables/...
```

I'd just run this across everything in there.